libres/lib/analysis/update.cpp

```diff
@@ -67,6 +66,46 @@ typedef struct {
         *node_info; /* mutable: For the serialization of one node */
 } serialize_info_type;
 
+class update_data_type {
```

Comment:
Considering adding docstring.
Something along the lines of "Container for all matrices needed for an update step"

Reply:
I have added it but it does not show up in the diff :shrug: